### DISCUSSION TITLE
To Home- Widget

### DISCUSSION THREAD

**Discussion by ElBartoxx** [organicmaps/organicmaps#8470] [Ideas]
First of all, thank you for the best map.

Please create a to-home-widget. **We users wont push on several buttons**. its better to push only one time to go home

And a function; **the nearest place to buy some food** (Restaurant, supermarket, snack bar). 
if you are on vacation in an unfamiliar place, are hungry and don't know where to buy food.

**Comment by biodranik** [comment]
Please follow #1133 for home button. Isn't it easier to search for "Food" category (or type `food` in the search) and see all nearby places in the list or on the map?
